src/components/Icons/CollectionRowActionIcon.js

```diff
@@ -41,22 +47,28 @@ const CollectionRowActionIcon = ({
       ActionSheetIOS.showActionSheetWithOptions({
         options: [
           'Cancel',
+          'Edit Collection',
+
           CollectionsDialogText[COLLECTIONS_DIALOG_ACTIONS.RENAME].title,
           CollectionsDialogText[COLLECTIONS_DIALOG_ACTIONS.DUPLICATE].title,
           CollectionsDialogText[COLLECTIONS_DIALOG_ACTIONS.DELETE].title,
         ],
-        destructiveButtonIndex: 3,
+        destructiveButtonIndex: 4,
         cancelButtonIndex: 0,
         userInterfaceStyle: 'dark',
       },
       (buttonIndex) => {
         if (buttonIndex === 0) {
           // cancel action
         } else if (buttonIndex === 1) {
+          selectCollectionAction(collectionId)
```

Comment:
This appears to be the `Edit Collection` option. If you can edit the name of the collection on that screen, then we shouldn't have the `rename` option below.